boards/sltb001a/Makefile.include

```diff
@@ -6,9 +6,16 @@ export CPU_MODEL = efr32mg1p132f256gm48
 PORT_LINUX ?= /dev/ttyACM0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
 
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
 # setup JLink for flashing
 export JLINK_DEVICE := EFR32MG1PxxxF256
 include $(RIOTMAKE)/tools/jlink.inc.mk
 
-# setup serial terminal
-include $(RIOTMAKE)/tools/serial.inc.mk
+# add board common drivers
+USEMODULE += boards_common_silabs
+USEMODULE += pic
```

Comment:
I wonder if this module name should be namespaced... I'd say to avoid confusions (pic can mean several things) better to name it `silabs_pic` or something similar?

Reply:
I'm fine with that. I first had `boards_common_silabs_pic`, but I found that too long.